GitHub/README.md

```diff
@@ -0,0 +1,44 @@
+## `git` and GitHub usage at Tidepool
+
+At Tidepool, we use `git` for version control, with repositories hosted [on GitHub](https://github.com/tidepool-org/ 'Tidepool on GitHub').
+
+This document describes our guidelines for `git` and GitHub usage.
+
+### Branches and pull requests
+
+In all application and library repositories, we use the `master` branch for production code. Production _releases_ are tied to git [tags](#tags).
+
+When making a change in any repository (whether developing a new feature or fixing a bug), our convention is to create a new branch with the name `<GitHub username>/<short description>` - e.g., `jebeck/new-signup-flow` or `jebeck/bugfix-form-validation`. The backend team sometimes/always(?) uses sprint-based branch names - e.g., `darinkrauss/sprint25` or `darinkrauss/sprint25-clinic-roles`.
+
+All significant changes should be made through a pull request that is approved by another Tidepool developer through a process of code review. We often use LGTM "looks good to me" as our approval sign-off at the conclusion of a review and/or one or more rounds of feedback and changes.
```

Comment:
I almost added something on the lines of "Or the "Ship It Squirrel" emoji. Perhaps I will ;)